public/app/features/scopes/ScopesService.ts

```diff
@@ -73,6 +73,11 @@ export class ScopesService implements ScopesContextValue {
     const scopeNodeId = queryParams.get('scope_node');
     // TODO: figure out when to remove this. scope_parent is for backward compatibility only
     const parentNodeId = queryParams.get('scope_parent');
+    const navigationScope = queryParams.get('navigation_scope');
```

Comment:
```suggestion
    const navigationScope = queryParams.get('nav_scope');
```
Maybe use this, as it is more compact? Also update the references for the generated links.